src/main/java/org/truffleruby/language/dispatch/CallDispatchHeadNode.java

```diff
@@ -43,7 +43,8 @@ private CallDispatchHeadNode(boolean ignoreVisibility, boolean onlyCallPublic, M
     }
 
     public Object call(VirtualFrame frame, Object receiver, Object method, Object... arguments) {
-        return dispatch(frame, receiver, method, null, arguments);
+        // TODO BJF REVIEW Refinments probably apply to these calls as well
+        return dispatch(frame, receiver, method, null, null, arguments);
```

Comment:
I think they don't, since these are calls in Java code, so typically in core methods, which are never in a refinement scope.